app/styles/layout/_footer.scss

```diff
@@ -0,0 +1,118 @@
+.footer{
+   font-family: $font-time-new-roman;
+   
+   @media (min-width:$px-1200){
+      margin: 0 auto;
+   }
+
+   &__baner{
+      text-align: center;
+
+      .img-banner{
+         &__img{
+            padding: 0;
```

Comment:
Làm theo conversion css (position > box model > color > text > orther)
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
đã fix